frontend/wallet/src/render/views/settings/SettingsPage.re

```diff
@@ -50,19 +49,28 @@ module Styles = {
       Theme.Text.Body.regular,
       style([
         userSelect(`none),
-        padding2(~v=`rem(1.), ~h=`rem(0.25)),
+        padding2(~v=`rem(1.), ~h=`rem(1.)),
         color(Theme.Colors.midnight),
         display(`flex),
         alignItems(`center),
-        hover([opacity(0.6)]),
         borderBottom(`px(1), `solid, Theme.Colors.slateAlpha(0.25)),
         lastChild([borderBottomWidth(`zero)]),
+        hover([
+          backgroundColor(Theme.Colors.midnightAlpha(0.05)),
+          selector(
+            "> :last-child",
```

Comment:
I don't understand this selector

Reply:
selects the last nested child of the wallet item (the span with chevron)

tried to use the classname .walletChevron in the selector, but couldn't get it to work - if you have another way of accomplishing this, then happy to switch it out